examples/sbol2/CreatingSBOL2Objects/Range.ipynb

```diff
@@ -0,0 +1,196 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using `Range` to Annotate a Sequence Region\n",
+    "\n",
+    "In this example, we will explore how to define continuous regions on a sequence using the `Range` location type in SBOL. The `Range` specifies a span of positions between a defined start and end point. This is useful for annotating features such as coding sequences or regulatory elements.\n",
+    "\n",
+    "We will demonstrate two `Range` annotations on a DNA sequence:\n",
+    "1. **Inline orientation**: A region from positions 5 to 20 on the forward strand.\n",
+    "2. **Reverse complement orientation**: A region from positions 25 to 41 on the reverse strand.\n",
+    "\n",
+    "This approach illustrates how SBOL makes it easy to annotate sequence features on both strands, highlighting the versatility of SBOL for standardized biological data representation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sbol2\n",
+    "from Bio.Seq import Seq  # Biopython for reverse complement\n",
+    "\n",
+    "# Create an SBOL document\n",
+    "doc = sbol2.Document()\n",
+    "\n",
+    "# Set a namespace for the document\n",
+    "sbol2.setHomespace('https://github.com/SynBioDex/SBOL-Notebooks')\n",
+    "\n",
+    "\n",
+    "sequence_elements = 'ATGCGTACGTAGCTAGTCTGATCGTAGCTAGTCGATGCAGGGC'\n",
+    "seq = sbol2.Sequence('example_sequence')\n",
+    "seq.elements = sequence_elements\n",
+    "seq.encoding = sbol2.SBOL_ENCODING_IUPAC\n",
+    "\n",
+    "# Add the sequence to the document\n",
+    "doc.addSequence(seq)\n",
+    "\n",
+    "# Create a ComponentDefinition for the sequence\n",
+    "comp_def = sbol2.ComponentDefinition('example_component', sbol2.BIOPAX_DNA)\n",
+    "comp_def.sequences = [seq.persistentIdentity]\n",
+    "\n",
+    "# Add the ComponentDefinition to the document\n",
+    "doc.addComponentDefinition(comp_def)\n",
+    "\n",
+    "# --- Range 1: Forward Strand (SBOL_ORIENTATION_INLINE) ---\n",
+    "# Create a SequenceAnnotation for a region on the forward strand\n",
+    "forward_annotation = sbol2.SequenceAnnotation('forward_region')\n",
+    "\n",
+    "# Define a Range from position 5 to 20 on the forward strand\n",
+    "forward_range = sbol2.Range('forward_range')\n",
```

Comment:
For a more compact example, you can include the start and end in the arguments and it will default to inline,
```
sbol2.Range('forward_range', start=5, end=20)
```

Reply:
Added in the latest commit 👍